pkg/deploy/lattice/target_group_synthesizer_test.go

```diff
@@ -152,7 +147,7 @@ func Test_SynthesizeTriggeredServiceExport(t *testing.T) {
 
 			assert.Nil(t, err)
 			tgName := latticestore.TargetGroupName(tt.svcExport.Name, tt.svcExport.Namespace)
-			dsTG, err := ds.GetTargetGroup(tgName, false)
+			dsTG, err := ds.GetTargetGroup(tgName, true)
 
```

Comment:
When create a serviceExport, we should expect LatticeDataStore have a serviceImport==true record for it